core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/footer.jelly

```diff
@@ -23,73 +23,65 @@ THE SOFTWARE.
+                    <div class="am-monitor__indicator-mobile"/>
                     <span class="am-monitor__count">
                         ${activeNonSecurityAMCount}
                     </span>
                 </a>
                 <div id="visible-am-list" class="am-list">
-                    <j:forEach var="am" items="${activeNonSecurityAM}">
-                        <div class="am-message">
-                            <st:include page="message.jelly" it="${am}" />
-                        </div>
-                    </j:forEach>
-                    <p style="text-align: center; margin: 10px 0 0 0;">
-                        <a href="${rootURL}/manage" onclick="document.location.href='${rootURL}/manage';">${%Manage Jenkins}</a>
-                    </p>
                 </div>
             </j:if>
         </div>
         <div id="visible-sec-am-container" class="am-container">
             <j:if test="${activeSecurityAMCount > 0}">
-                <a id="visible-sec-am-button" class="am-button security-am" href="#" onclick="toggleVisibleSecAmList(event)" title="${%tooltipSec(activeSecurityAMCount)}">
-                    <l:svgIcon href="${resURL}/images/material-icons/svg-sprite-action-symbol.svg#ic_warning_shield_24px" />
-                    <div class="am-monitor__indicator-mobile" />
+                <a id="visible-sec-am-button"
+                   class="am-button security-am"
+                   href="#"
+                   data-href="${rootURL}/administrativeMonitors/securityPopupContent"
+                   title="${%tooltipSec(activeSecurityAMCount)}">
+                    <l:svgIcon
+                            href="${resURL}/images/material-icons/svg-sprite-action-symbol.svg#ic_warning_shield_24px"/>
+                    <div class="am-monitor__indicator-mobile"/>
                     <span class="am-monitor__count">
                         ${activeSecurityAMCount}
                     </span>
                 </a>
                 <div id="visible-sec-am-list" class="am-list">
-                    <j:forEach var="am" items="${activeSecurityAM}">
-                        <div class="am-message">
-                            <st:include page="message.jelly" it="${am}" />
-                        </div>
-                    </j:forEach>
-                    <p style="text-align: center; margin: 10px 0 0 0;">
-                        <a href="${rootURL}/manage" onclick="document.location.href='${rootURL}/manage';">${%Manage Jenkins}</a>
-                    </p>
                 </div>
             </j:if>
         </div>
         <script type="text/javascript">
             (function() {
-                var amContainer = document.getElementById("visible-am-container");
-                var amInsertion = document.getElementById("visible-am-insertion");
+            var amContainer = document.getElementById("visible-am-container");
```

Comment:
looks like the indent change should be reverted here?

Reply:
YES